src/components/FolderList.js

```diff
@@ -0,0 +1,66 @@
+import "./FolderList.css";
+import noImage from "../assets/noImage.svg";
+
+function formatDate(value) {
+  const date = new Date(value);
+  return `${date.getFullYear()}. ${date.getMonth() + 1}. ${date.getDate()}`;
+}
+
+function formatTimeDifference(value) {
+  const now = new Date();
+  const diffInMilliseconds = now - new Date(value);
+
+  const units = [
+    { label: "year", duration: 365 * 24 * 60 * 60 * 1000 },
+    { label: "month", duration: 30 * 24 * 60 * 60 * 1000 },
+    { label: "day", duration: 24 * 60 * 60 * 1000 },
+    { label: "hour", duration: 60 * 60 * 1000 },
+    { label: "minute", duration: 60 * 1000 },
+  ];
+
+  for (const unit of units) {
+    const diffInUnit = Math.floor(diffInMilliseconds / unit.duration);
+
+    if (diffInUnit === 1) {
+      return `1 ${unit.label} ago`;
+    } else if (diffInUnit > 1) {
+      return `${diffInUnit} ${unit.label}s ago`;
+    }
+  }
+}
+
+function FolderListItem({ item }) {
+  const { title, createdAt, url, description, imageSource } = item;
+
+  return (
+    <a href={url} target="_blank" rel="noopener noreferrer">
+      {imageSource ? (
+        <img src={imageSource} className="card-box-img" alt="" />
+      ) : (
+        <img src={noImage} className="card-box-img" alt="" />

```

Comment:
이 부분은 `src`만 바꿔주면 되죠? 
```suggestion
   const src = imageSource ?? noImage
   <img src={src} className="card-box-img" alt="" />
```